wiki/Unofficial-search-plugins.mediawiki

```diff
@@ -652,11 +640,17 @@ The minimum supported Python version (across all platforms) is specified at [htt
 | [https://gitlab.com/CrimsonKoba/qb-search-plugin/-/raw/master/unionfansub.py [[https://raw.githubusercontent.com/Pireo/hello-world/master/Download.gif]] ]<br /> [https://gitlab.com/CrimsonKoba/qb-search-plugin [[https://github.com/Pireo/hello-world/blob/master/Help%20book.gif]] ]
 | ✔ qbt 4.5.x / python 3.+
 |-
-
-| [[https://raw.githubusercontent.com/CravateRouge/qBittorrentSearchPlugins/master/yggtorrent.png]] [https://yggtorrent.fi/ YggTorrent]
+| [[https://raw.githubusercontent.com/CravateRouge/qBittorrentSearchPlugins/master/yggtorrent.png]] [https://www.ygg.re/ YggTorrent]
 | [https://github.com/CravateRouge/qBittorrentSearchPlugins CravateRouge]
 | 1.0
 | 25/Nov<br />2019
 | [https://raw.githubusercontent.com/CravateRouge/qBittorrentSearchPlugins/master/yggtorrent.py [[https://raw.githubusercontent.com/Pireo/hello-world/master/Download.gif]] ]<br /> [https://github.com/CravateRouge/qBittorrentSearchPlugins/blob/master/README.md [[https://github.com/Pireo/hello-world/blob/master/Help%20book.gif]] ]
 | ❗ Working when DDOS cloudfare<br />protection is disabled
+|-
+| [[https://raw.githubusercontent.com/Laiteux/YggAPI-qBitTorrent-Search-Plugin/refs/heads/main/yggapi.ico]] [https://yggapi.eu/ YggAPI]
+| [https://github.com/Laiteux/YggAPI-qBitTorrent-Search-Plugin Laiteux]
+| 1.1
+| 31/Jan<br />2025
+| [https://raw.githubusercontent.com/Laiteux/YggAPI-qBitTorrent-Search-Plugin/refs/heads/main/yggapi.py [[https://raw.githubusercontent.com/Pireo/hello-world/master/Download.gif]] ]<br /> [https://github.com/Laiteux/YggAPI-qBittorrent-Search-Plugin/blob/main/README.md [[https://github.com/Pireo/hello-world/blob/master/Help%20book.gif]] ]
```

Comment:
https://raw.githubusercontent.com/Laiteux/YggAPI-qBitTorrent-Search-Plugin/refs/heads/main/yggapi.py

Is there a need for `refs/heads` here too??